website/docs/d/stream_connection.html.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: stream connection"
+sidebar_current: "docs-mongodbatlas-datasource-stream-connection"
+description: |-
+    Describes a Stream Connection.
+---
+
+# Data Source: mongodbatlas_stream_connection
+
+`mongodbatlas_stream_connection` describes a stream connection.
+
+-> **NOTE:** Atlas Streams is currently in Preview, you'll need to set the environment variable `MONGODB_ATLAS_ENABLE_BETA=true` to use this data source. To learn more about stream processing and existing limitations, see the [Atlas Stream Processing Documentation](https://www.mongodb.com/docs/atlas/atlas-sp/overview/#atlas-stream-processing-overview).
```

Comment:
```suggestion
-> **NOTE:** Atlas Stream Processing is currently in Private Preview, you'll need to set the environment variable `MONGODB_ATLAS_ENABLE_BETA=true` to use this data source. To learn more about stream processing and existing limitations, see the [Atlas Stream Processing Documentation](https://www.mongodb.com/docs/atlas/atlas-sp/overview/#atlas-stream-processing-overview).
```

Reply:
will not include the Private part as this will change in January